views/login/sin.pug

```diff
@@ -7,14 +7,21 @@ block content
 
   h1 #{title}
 
+  if data.personal
+    h2 Thanks, #{data.personal.firstName} #{data.personal.lastName}!
+
   p Now that we have your personal code, we need some additional information in order to verify that it’s you.
 
   p Please enter your Social Insurance Number (SIN) in the field below.
 
   form.cra-form(method='post')
     div
       label(for='sin') Social Insurance Number
-      span.cra-form-message For Example: 123 456 789
+      if data.personal
+        span.cra-form-message For Example: #{SINFilter(data.personal.sin)}
```

Comment:
I don't know why my brain didn't realize I could pull functions in directly to pug templates, but this is nice 👌

Reply:
I also didn't know this!

Reply:
Yeah, Pug's way of doing "filters" only works for static text, so this is probably what we want most of the time.